### PR TITLE
Recommend WinGet

### PR DESCRIPTION
**Pull request by mgreenegit**
# PR Summary

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/docs-conceptual/install/Installing-PowerShell-on-Windows.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ title: Installing PowerShell on Windows
 There are multiple ways to install PowerShell in Windows. Each install method is designed to support
 different scenarios and workflows. Choose the method that best suits your needs.
 
-- [MSI package](#msi) - The best choice for the everyday user.
-- [ZIP package](#zip) - The easiest way to install multiple versions or use in a CI/CD pipeline.
+- [Winget](#winget) - Recommended way to install PowerShell in Windows.
```
Comment (sdwheeler) [inline reference/docs-conceptual/install/Installing-PowerShell-on-Windows.md:11]:
```suggestion
- [Winget](#winget) - Recommended way to install PowerShell in Windows clients.
```

File: reference/docs-conceptual/install/Installing-PowerShell-on-Windows.md [hunk 2]
```diff
@@ -30,6 +30,37 @@ different scenarios and workflows. Choose the method that best suits your needs.
 > Download links for every package are found in the **Assets** section of the Release page. The
 > **Assets** section may be collapsed, so you may need to click to expand it.
 
+## <a id="winget" />Install PowerShell using Winget (recommended)
+
+Winget, the Windows Package Manager, is a command-line tool enables developers to discover, install,
+upgrade, remove, and configure applications on Windows 10 computers. This tool is the client
```
Comment (sdwheeler) [inline reference/docs-conceptual/install/Installing-PowerShell-on-Windows.md:36]:
```suggestion
upgrade, remove, and configure applications on Windows client computers. This tool is the client
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
I am not sure about this recommendation
- Winget is not installed in Windows by default
- Winget is not usable on Windows Server skus
- The MSI package works on all versions and skus of Windows

**Comment by mgreenegit** [comment]
#sign-off

**Comment by mgreenegit** [comment]
#sign-off

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**